share/completions/jest.fish

```diff
@@ -0,0 +1,79 @@
+complete -f -c jest -l modulePaths -d 'An alternative API to setting the NODE_PATH env variable'
+complete -f -c jest -l noStackTrace -d 'Disables stack trace in test results output'
+complete -f -c jest -l notify -d 'Activates notifications for test results'
+complete -f -c jest -l onlyChanged -s o -d 'Run tests based on the changed files'
+complete -c jest -l outputFile -d 'A file where to write test results'
+complete -c jest -l preset 'A preset that is used as a base for Jest''s configuration'
+complete -f -c jest -l projects -d 'A list of projects that use Jest to run all tests'
+complete -f -c jest -l reporters -d 'A list of custom reporters for the test suite'
+complete -f -c jest -l resetMocks -d 'Automatically reset mock state between every test'
+complete -f -c jest -l resetModules -d 'Whatever to reset the module registry for every test'
+complete -f -c jest -l resolver -d 'A JSON string which allows the use of a custom resolver'
+complete -c jest -l rootDir -d 'The root directory that Jest should scan for tests and modules within'
+complete -c jest -l roots -d 'Paths to directories that Jest should use to search for files in'
+complete -f -c jest -l runInBand -s i -d 'Run all tests serially in the current process'
+complete -c jest -l setupFiles -d 'The paths to modules to configure the testing environment'
+complete -c jest -l setupTestFrameworkScriptFile -d 'The path to a module to configure the testing framework'
+complete -f -c jest -l showConfig -d 'Print your jest config and then exits'
+complete -f -c jest -l silent -d 'Prevent tests from printing messages through the console'
+complete -c jest -l snapshotSerializers -d 'Snapshot serializer modules Jest should use for snapshot testing'
+complete -f -c jest -l testEnvironment -d 'Alias for --env'
+complete -f -c jest -l testMatch -d 'The glob patterns Jest uses to detect test files'
+complete -f -c jest -l testNamePattern -s t -d 'Run only tests with a name that matches the regex pattern'
+complete -f -c jest -l testPathIgnorePatterns 'Paths to skip'
+complete -f -c jest -l testPathPattern -d 'Regex for test paths'
+complete -f -c jest -l testRegex -d 'The regexp pattern Jest uses to detect test files'
+complete -f -c jest -l testResultsProcessor -d 'Allows the use of a custom results processor'
+complete -f -c jest -l testRunner -d 'Allows to specify a custom test runner'
+complete -f -c jest -l testURL -d 'This option sets the URL for the jsdom environment'
+complete -f -c jest -l timers -d 'Setting this value to fake value of timers'
+complete -f -c jest -l transform -d 'A JSON string which maps from regular expressions to paths to transformers'
+complete -f -c jest -l transformIgnorePatterns -d 'Regexp patterns that are matched against all source file paths before transformation'
+complete -f -c jest -l unmockedModulePathPatterns -d 'a regex pattern of module to unmock'
+complete -f -c jest -l updateSnapshot -s -u -d 'Use this flag to re-record snapshots'
```

Comment:
It is the short option for updateSnapshot. I forgot to remove '-'. I also forgot to add -d for some options. I fixed it.